components/footer.js

```diff
@@ -14,9 +16,22 @@ export default function Footer() {
             <a href="#contact">Contact us</a>
           </p>
         </div>
-        <p className="body2-bold">
-          <a href="mailto:tej.fellowship@gmail.com">tej.fellowship@gmail.com</a>
-        </p>
+        <div className="flex gap-small">
+          <a
```

Comment:
<img width="394" alt="image" src="https://github.com/tech-equity-justice/tech-equity-justice.github.io/assets/5488235/b2dd1121-ab83-40d6-83e7-22e18dbae807">
These links don't show properly in mobile view